package.json

```diff
@@ -8,16 +8,18 @@
   ],
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@1.2.3",
-    "@edx/frontend-component-footer": "13.0.2",
+    "@edx/frontend-component-footer": "13.0.3",
     "@edx/frontend-enterprise-catalog-search": "10.5.0",
     "@edx/frontend-enterprise-hotjar": "6.0.0",
     "@edx/frontend-enterprise-logistration": "8.0.0",
     "@edx/frontend-enterprise-utils": "8.0.0",
+    "@edx/frontend-logging": "^4.3.0",
```

Comment:
[inform] this needs to be removed from this file. it a private NPM package installed only by 2U's GoCD build+deploy MFE pipeline.

Reply:
[nit] does this still need to be either pinned/removed?